src/main/java/gregtech/api/recipes/RecipeBuilder.java

```diff
@@ -887,6 +893,32 @@ public R copy() {
         return (R) new RecipeBuilder<>(this);
     }
 
+    /**
+     * Only use if you absolutely don't want the recipe to be run through any build actions.
+     * Instead, you should blacklist specific actions with {@link #ignoreBuildAction(ResourceLocation)}
+     */
+    public R ignoreAllBuildActions() {
+        this.ignoreAllBuildActions = true;
+        return (R) this;
+    }
+
+    public R ignoreBuildAction(ResourceLocation buildActionName) {
+        if (ignoredBuildActions == null) {
+            ignoredBuildActions = new Object2ObjectOpenHashMap<>();
+        } else if (!recipeMap.getBuildActions().containsKey(buildActionName)) {
+            GTLog.logger.error("Recipe map {} does not contain build action {}!", recipeMap, buildActionName,
+                    new Throwable());
+            return (R) this;
+        } else if (ignoredBuildActions.containsKey(buildActionName)) {
+            GTLog.logger.error("Builder already ignores {}!", buildActionName, new Throwable());
```

Comment:
I'm not sure we want to log in this case, since it doesn't actually cause problems. Might make some logic easier by not having to avoid and work around the logged error.